src/controllers/auth.controller.ts

```diff
@@ -33,7 +33,7 @@ export class AuthController {
     @BodyParam('email') email: string,
     @BodyParam('password') password: string
   ) {
-    if (!email || !password) {
+    if (!this.usersService.credentialsExists(email, password)) {
```

Comment:
What about renaming this like `givenCredentials`? Exists makes me thing we are checking that they appear on the DB, wdyth?